doc/release/RELEASE-NOTES.md

```diff
@@ -10,6 +10,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * Release date: September 7, 2024
 
 ### Code/API changes
+* [OSDEV-1167](https://opensupplyhub.atlassian.net/browse/OSDEV-1167) - Search. Update field names in Open Search. The following parameter/field names in the API schema has been changed:
```

Comment:
Could you please specify the name of the API endpoint you updated? Right now, it is unclear exactly where you made these changes.

Reply:
Added endpoint name -  for GET api/v1/production-locations